src/content-scripts/canvas.js

```diff
@@ -54,6 +57,31 @@ function toggleCanvasOff(){
   console.log('canvas removed!');
 };
 
+// // utlity function to save serialized canvas data
+// function saveCanvas(){
+
+// };
+
+function serializeOut = function() {
```

Comment:
Storing data in local storage might be a good way to test serialization and deserialization for now.

However, at some point, we'll definitely need to think about how we will store it in Firebase and how we will structure our Firebase data.